<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMaintenance</class>
 <widget class="QMainWindow" name="AdminMaintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add A New City</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_city_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
text-align: center;
color: white;
font-size: 14px;
background-color: rgb(176, 23, 255);
border: 1px solid black;
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAddNewCity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove A City</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRemoveCity">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_city_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
text-align: center;
color: white;
font-size: 14px;
background-color: rgb(176, 23, 255);
border: 1px solid black;
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxUpdateFoods">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Traditional Foods Prices</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_update_food">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: #7cc8b1;
text-align: center;
color: white;
font-size: 10px;
border: 1px solid white;
border-radius: 5px;
margin: 10px;
padding: 10px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_food_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
text-align: center;
color: white;
font-size: 14px;
background-color: rgb(176, 23, 255);
border: 1px solid black;
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_update_food_price">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove A Food</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_remove_food">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: #7cc8b1;
text-align: center;
color: white;
font-size: 10px;
border: 1px solid white;
border-radius: 5px;
margin: 10px;
padding: 10px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_food_button">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
text-align: center;
color: white;
font-size: 14px;
background-color: rgb(176, 23, 255);
border: 1px solid black;
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New Food</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_food_button">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>610</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
text-align: center;
color: white;
font-size: 14px;
background-color: rgb(176, 23, 255);
border: 1px solid black;
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_add_food">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>400</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
background-color: #7cc8b1;
text-align: center;
color: white;
font-size: 10px;
border: 1px solid white;
border-radius: 5px;
margin: 10px;
padding: 10px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_purchase_history_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
text-align: center;
color: white;
font-size: 14px;
background-color: rgb(176, 23, 255);
border: 1px solid black;
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Delete Purchase History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RefreshpushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
text-align: center;
color: white;
font-size: 14px;
background-color: rgb(0, 214, 255);
border: 1px solid black;
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Refresh Page</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
